usr/libexec/greenboot/greenboot-status

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+if [ ! -f /run/motd.d/boot-status ]; then
+    if [ ! -d /run/motd.d ]; then
+        mkdir /run/motd.d
+    fi
+    touch /run/motd.d/boot-status
```

Comment:
On creating files in `/run/motd.d/*`, it occurred to me that since any package can write here, packages should own the file that they write there in the specfile (with a tmpfile to create the `/run/motd.d/mypkgmotd` file). I'm planning to do this with the `/run/motd.d/console-login-helper-messages.motd` that gets generated ([rough issue for this](https://github.com/rfairley/console-login-helper-messages/issues/24)).

Doing the check `if [ ! -f /run/motd.d/boot-status ]; then` already makes this safe though, so I don't think the ownership of `/run/motd.d/boot-status` should block this PR. Just something to keep in mind.